platform/viewer/cypress/support/commands.js

```diff
@@ -30,12 +30,73 @@ Cypress.Commands.add('openStudy', (patientName) => {
     cy.visit('/');
     cy.get('#patientName')
       .type(patientName);
-    cy.get('.studylistStudy > .patientName')
-      .contains(patientName)
-      .click();
+
+    const patientResult = '.studylistStudy > .patientName';
+
+    cy.get(patientResult)
+      .then({ timeout: 5000 }, ($patientResult) => {
+        cy.contains(patientName)
+        .click();
+      })
   }
 );
 
 Cypress.Commands.add('drag', {prevSubject: 'element',},
   (...args) => DragSimulator.simulate(...args)
 );
+
+// Command to perform two clicks into two different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addLine', (element, initPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
+    .then(() =>{
+      cy.get(element)
+        .trigger('mousemove', { clientX: finalPosition[0], clientY: finalPosition[1] })
+        .click({ force: true })
+    })
+});
+
+
+// Command to perform three clicks into three different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addAngle', (element, initPosition, midPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
+    .then(() =>{
+      cy.get(element)
+        .trigger('mousemove', { clientX: midPosition[0], clientY: midPosition[1] })
+        .click(midPosition[0], midPosition[1], { force: true })
+    }).then(() =>{
+      cy.get(element)
+        .trigger('mousemove', { clientX: finalPosition[0], clientY: finalPosition[1] })
+        .click(finalPosition[0], finalPosition[1], { force: true })
+    })
+});
+
+
+//Command to wait DICOM image to load into the viewport
+Cypress.Commands.add('waitDicomImage', (timeout = 10000) => {
+  // Give an alias to request
+  cy.server().route("GET", '/dcm4chee-arc/aets/DCM4CHEE/rs/studies/**').as('imageRequest')
+ 
+  // Wait for response.status to be 200
+  cy.wait('@imageRequest', {timeout:timeout}).its('status').should('be', 200) 
+  Cypress.on('uncaught:exception', (err, runnable) => {
+    cy.log('No Image Request was made.');
+    // returning false here prevents Cypress from failing the test
+    return false
+  })
+});
+
+
+//Command to reset the viewport changes throught the cornerstone method
+Cypress.Commands.add('resetViewport', () => {
+  cy.window()
+    .its('cornerstone')
+    .then(($cornerstone) => {
+      $cornerstone.reset($cornerstone.getEnabledElements()[0].element)
+    });
+});
```

Comment:
This is why I hinted that we should probably use the UI to reset the viewport.
At first glance, this feels like a great solution, but it fails in a few places that the UI button should not:

- There's no guarantee that `cornerstone` will be available on the window. We expose it now for convenience/debugging, but may remove it in the future.
- Resetting a VTK (or other) viewport does not use this logic
- If you have multiple cornerstone viewports, this would only reset the first (not the active one)
- There's no guarantee that `getEnabledElements` ordering corresponds to the UIs viewport ordering

Reply:
Replaced command to use the UI to reset the viewport